packages/components/dialog/src/dialog.stories.ts

```diff
@@ -1,129 +1,121 @@
-      }
+    subtitle: 'Subtitle'
+  }
+};
 
-      sl-button-bar:not([reverse]) > sl-button:first-child {
-        margin-right: auto;
-      }
+export const CloseButton: Story = {};
 
-      @media screen and (max-width: 600px) {
-        sl-button-bar {
-          align-items: stretch;
-          flex-direction: column-reverse;
-          flex-grow: 1;
-          flex-shrink: 0;
-        }
+export const DisableCancel: Story = {
+  args: {
+    body: 'You cannot close me by pressing the Escape key, or clicking the backdrop. This dialog also has no close button. The only way to close it is by clicking one of the action buttons.',
+    closeButton: false,
+    disableCancel: true,
+    maxWidth: '300px',
+    title: 'Disable cancel and close button'
+  }
+};
 
-        sl-button-bar[reverse] > sl-button:first-child {
-          margin-left: inherit;
+export const FooterButtons: Story = {
+  args: {
+    footerButtons: () => html`
+      <style>
+        sl-dialog::part(footer-bar) {
+          --sl-button-bar-align: space-between;
         }
-
-        sl-button-bar:not([reverse]) > sl-button:first-child {
-          margin-right: inherit;
+        @media (min-width: 600px) {
+          sl-button:first-of-type {
+            margin-inline-end: auto;
+          }
         }
-      }
-    </style>
-    <sl-button fill="outline" @click=${onClick}>Show Dialog</sl-button>
-    <sl-dialog close-button>
-      <span slot="subtitle">Dialog subtitle</span>
-      <span slot="title">Dialog title</span>
-      <div>You cannot scroll the body once the dialog is open.</div>
-      <sl-button-bar slot="footer" align="space-between" .reverse=${reverse}>
-        <sl-button fill="ghost" variant="default" sl-dialog-close autofocus>Cancel</sl-button>
-        <sl-button fill="outline" variant="primary" sl-dialog-close>Action 2</sl-button>
-        <sl-button fill="solid" variant="primary" sl-dialog-close>Action</sl-button>
-      </sl-button-bar>
-    </sl-dialog>
-  `
+      </style>
+      <sl-button fill="ghost" slot="actions" variant="default" sl-dialog-close autofocus>Cancel</sl-button>
+      <sl-button fill="outline" slot="actions" variant="primary" sl-dialog-close>Action 2</sl-button>
+      <sl-button fill="solid" slot="actions" variant="primary" sl-dialog-close>Action</sl-button>
+    `,
+    title: 'Dialog with extra footer buttons'
+  }
 };
 
-export const DisableCancel: StoryObj = {
-  render: () => html`
-    <sl-button fill="outline" @click=${onClick}>Show Dialog</sl-button>
-    <sl-dialog disable-cancel close-button align="space-between">
-      <span slot="title">Disable cancel</span>
-      <p>You cannot close me by pressing the Escape key, or clicking the backdrop.</p>
-      <sl-button slot="actions" fill="solid" variant="default" sl-dialog-close autofocus>Action 2</sl-button>
-      <sl-button slot="actions" fill="solid" variant="primary" sl-dialog-close>Action</sl-button>
-    </sl-dialog>
-  `
+export const HeaderButtons: Story = {
```

Comment:
Can you please also show here that the close button can have the same styling as the default, but a different icon inside? As far as I remember that was the case, that it can have a different icon in the closing button :) and other ghost icon buttons next to this one.

<img width="1567" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/dcc78e3d-22b8-47ef-952a-2e4d27535eda">

Reply:
Added